sharness/__snapshots__/test-instance/config/plugins/sourcecred/ethereum/ethereumAddresses.json

```diff
@@ -0,0 +1 @@
+[{"type":"sourcecred/ethAddressFile","version":"0.0.1"},["0x2Ccc7cD913677553766873483ed9eEDdB77A0Bb0", "0xb4124cEB3451635DAcedd11767f004d8a28c6eE7","0x8401Eb5ff34cc943f096A32EF3d5113FEbE8D4Eb"]]
```

Comment:
As mentioned, you can pass `, {space: 4}` to `json-stable-stringify` to
get it to spit out a diff-friendly format.